testing/src/cypress/page-objects/Institution-objects/DashboardInstitutionObject.ts

```diff
@@ -1,5 +1,98 @@
 import BaseMethods from "./BaseMethods";
 
+export enum CredentialTypes {
+  UndergraduateCertificate = "undergraduateCertificate",
+  UndergraduateDiploma = "undergraduateDiploma",
+  UndergraduateDegree = "undergraduateDegree",
+  GraduateCertificate = "graduateCertificate",
+  GraduateDiploma = "graduateDiploma",
+  GraduateDegreeOrMasters = "graduateDegreeOrMasters",
+  PostgraduateOrDoctorate = "postGraduateOrDoctorate",
+  QualifyingStudies = "qualifyingStudies",
+}
+
+export enum ProgramLengthOptions {
+  WeeksToLessThanYear = "12WeeksToLessThan1Year",
+  OneToTwoYears = "1YearToLessThan2Years",
+  TwoToThreeYears = "2YearsToLessThan3Years",
+  ThreeToFourYears = "4YearsToLessThan5Years",
```

Comment:
`ThreeToFourYears = "4YearsToLessThan5Years",`, LHS and RHS are not matching.

Reply:
My bad, missed it.